modules/apps/commerce/commerce-wish-list-service/src/main/java/com/liferay/commerce/wish/list/service/impl/CommerceWishListServiceImpl.java

```diff
@@ -39,12 +39,11 @@ public CommerceWishList addCommerceWishList(
 			String name, boolean defaultWishList, ServiceContext serviceContext)
 		throws PortalException {
 
-		if (getUserId() != serviceContext.getUserId()) {
-			_checkManagePermission(serviceContext.getScopeGroupId());
-		}
+		_checkManagePermission(serviceContext.getScopeGroupId());
```

Comment:
Check on userId is necessary here to allow a buyer to create his wish list without having permission to manage all wishlists on a site.